test/pummel/test-regress-GH-814.js

```diff
@@ -46,7 +46,7 @@ var timeToQuit = Date.now() + 8e3; //Test during no more than this seconds.
     }
   }
   else {
-    throw Error("Buffer GC'ed test -> FAIL");
+    throw new Error("Buffer GC'ed test -> FAIL");
```

Comment:
While you're here, can you change to single quotes on this line? There is one other line in this file that uses double quotes, but it might not be appropriate to change in this PR.